docs/guides/join_network.md

```diff
@@ -48,7 +48,7 @@ This document outlines the steps to join the public testnet hosted by [Chainsafe
 6. Start the RPC server
 
     ```bash
-    ethermintcli rest-server --laddr "tcp://localhost:8545" --unlock-key $KEY --chain-id etherminttestnet-777 --trace
+    ethermintcli rest-server --laddr "tcp://localhost:8545" --unlock-key $KEY --chain-id etherminttestnet-777 --trace --rpc-api "web3,eth,net,personal"
```

Comment:
why are people turning on personal?

Reply:
because they can have their own account unlocked if they wish or use any of the personal api methods